### PR TITLE
[MNG-5693] Change logging of MojoExceptions to console

### PR DESCRIPTION
**Pull request by slachiewicz**

### CODE REVIEW

File: maven-core/src/main/java/org/apache/maven/exception/DefaultExceptionHandler.java [hunk 1]
```diff
@@ -124,7 +115,7 @@ private ExceptionSummary handle( String message, Throwable exception )
             message = getMessage( message, exception );
         }
 
-        return new ExceptionSummary( exception, message, reference, children );
+        return new ExceptionSummary( exception, message, null, children );
```
Comment (eolivelli) [inline maven-core/src/main/java/org/apache/maven/exception/DefaultExceptionHandler.java]: Are we losing the exception at all?

**Review by eolivelli — CHANGES_REQUESTED** [review]

File: maven-core/src/main/java/org/apache/maven/exception/DefaultExceptionHandler.java (hunk 1 above)
Comment (slachiewicz) [reply maven-core/src/main/java/org/apache/maven/exception/DefaultExceptionHandler.java]: No, in reference we store shot exception name or links to cwiki

Comment (eolivelli) [reply maven-core/src/main/java/org/apache/maven/exception/DefaultExceptionHandler.java]: Got it
Why not dropping the field?

Comment (rfscholte) [reply maven-core/src/main/java/org/apache/maven/exception/DefaultExceptionHandler.java]: I think it should be possible for some Maven Exceptions to provide their own reference, so plugins and extensions can refer to their own documentation. This cleanup looks a bit too agressive to me.

**Comment by jira-importer** [comment]
Resolve #6616

**Pull request closed without merging**